### ISSUE TITLE
Ctrl + Right Click triggers context menu and jumps to definition

### ISSUE THREAD

**Issue by htmlHxcker** [microsoft/vscode#147762]
Issue Type: <b>Bug</b>

1. Open any file in the editor
2. Ctrl + Right Click on any function or import that is defined in another file

**Actual Result**
The editor shows the context menu for a split second then jumps to the definiton of the file in a new tab.

**Expected Result**
To be honest, I don't really know what to expect. I was trying to peek a definition and then misclicked before I noticed this. I suppose jumping to he definition without the context menu would be OK.

VS Code version: Code 1.66.2 (dfd34e8260c270da74b5c2d86d61aee4b6d56977, 2022-04-11T07:46:01.075Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No
Remote OS version: Linux x64 5.10.16.3-microsoft-standard-WSL2

**Comment by jrieken** [comment]
I cannot reproduce... @htmlHxcker Do you use native or custom menus? Is it possible that the context menu shows perfectly under the mouse and that the entry that under the mouse is triggered?

**Comment by htmlHxcker** [comment]
I can still reproduce this though, here's a short clip showing the bug in action 

https://user-images.githubusercontent.com/64586121/167211325-7baddb3a-3ae4-4e17-bd6c-b1cdcb8027e6.mp4

**Comment by jrieken** [comment]
The seems broken. Please reupload

**Comment by htmlHxcker** [comment]
That's weird, it works fine for me across browsers. Can you view it now? @jrieken 
https://user-images.githubusercontent.com/64586121/167390798-d94534b7-8db1-4729-a2d0-f1de6cb98286.mp4

**Comment by jrieken** [comment]
Yeah, funny. It doesn't seem to work on Safari but with Edge/Chrome...

**Comment by aeschli** [comment]
Verified on Edge/Windows
